flycheck-hledger.el

```diff
@@ -105,11 +105,17 @@ https://hledger.org/hledger.html#check."
     (one-or-more (or (seq (one-or-more digit) " ") (>= 2 " ")) "| " (zero-or-more nonl) "\n")                   ; excerpt lines
     (message "\n" (one-or-more bol (zero-or-more nonl) "\n")))
 
+   ;; hledger 1.31+ error with LINE and context:
+   (error bol "hledger" (optional ".exe") ": Error: " (file-name (optional alpha ":") (+ (not ":"))) ":" line ":\n"
+    (>= 2 (optional "\n") (one-or-more (or (seq (one-or-more digit) " ") (>= 2 " ")) "| " (zero-or-more nonl) "\n")) ; excerpt lines
+    "\n"
+    (message (one-or-more (zero-or-more nonl) (or "\n" string-end)))))
```

Comment:
Sorry! I’d been hunting for this parenthesis because something looked off to me but I just couldn’t see that it was at the end of the previous block.

Your comment about the tests made me realize the error pattern I added subsumes the other one.